pkg/apis/workspace/util.go

```diff
@@ -0,0 +1,5 @@
+package workspace
+
+// GenericConfig is a generic model to describe config which shields the difference among multiple concrete
+// models. GenericConfig is designed for extensibility, used for module, terraform runtime config, etc.
+type GenericConfig map[string]any
```

Comment:
Why is it preferable to define it in the util file? It would be more appropriate to rename `util` to `type`.